Tasks/IISWebAppDeployment/task.json

```diff
@@ -284,13 +284,13 @@
             "visibleRule": "AppPoolIdentity = SpecificUser"
         },
         {
-            "name": "AppCmdArgs",
-            "type": "string",
-            "label": "Additional Appcmd.exe Arguments",
+            "name": "AppCmdCommands",
```

Comment:
Our Compat tests might fail, since you are editing name. While it might be ok to edit this as we haven't shipped task yet, our Compat test today compares tasks between last sprint and this sprint tasks and could fail.